### ISSUE TITLE
Questions for Hannah

### ISSUE THREAD

**Issue by teenie-quaggard** [GSG-G7/mental-snapp#3]
@MentalSnapp Hi again Hannah!
I just got off the phone with the team.  Just to give a bit of a debrief:

1. They have decided that they can fit authentication into the scope of the project. In terms of stack, they will use Auth0 and Firebase (this adds a number of new pages and functionality to the app, so I will be revisiting our user stories/pages before Sunday).

2. We've noticed that there hasn't been any work on the Figma. Don't worry, the team is totally happy to design something for you if you haven't got the time this week! One thing is that it may be confusing if both you and them are designing at the same time and so I just wanted to reiterate that your last chance to create any templates or layouts for the app will be Sunday. After that, the team will pick up from where ever you have left off and will finish the designs. 

3. In order for the team to design they will need the color palette, fonts and logos from you! Please try and get these to me by tomorrow if possible. 

Thanks Hannah! Please let me know if you have any issues!

**Comment by MentalSnapp** [comment]
Thanks Kristina!

I’m going to make a start this afternoon/tomorrow. We’re thinking our way
through names and branding at the moment. Busy time and lots of decisions
to make!

Will be on this later on.

V best
Hannah

**Comment by teenie-quaggard** [comment]
Hi Hannah, 
Thank you for sending all the info over. The team will now take over and design the pages that you did not. I noticed that you tried something other than we had spoken about for the display of the log of entries. I just wanted to be sure that you are happy with this design as finalised. Please confirm! Thank you!

**Comment by MentalSnapp** [comment]
Yes I’m not sure about my design in terms of font size and accessibility so
I’m open to other ideas. All my designs are sketches to work from.

Thanks so much.

**Comment by teenie-quaggard** [comment]
Good to know, thanks Hannah!

**Comment by teenie-quaggard** [comment]
Hi Hannah @MentalSnapp ! I hope you are well! I have just got off from a call with the team, and the design is looking really great! 

I just wanted to touch base and see if you were able to find any users to test the prototype between 9-11am on Wednesday.

Just to give you a bit of background, the users are encouraged to attend the 30 minute user testing remotely (they will need a phone and laptop with internet connection). I will ask the users to complete some basic tasks and you will take notes on things that the users were confused by or had problems finding. I will make a script which I can send you later today.

Just so you know, I will have to leave the user testing immediately at 11am as we are having a Founders and Coders event!

**Comment by MentalSnapp** [comment]
Thanks Kristina!

Yes I’ve got 3 people but I thought they had to attend the offices and I’ve
offered them hourly payment. Is it only 30 minutes then by remote?

**Comment by teenie-quaggard** [comment]
Hi @MentalSnapp - they are totally welcome to come in to the office! It's only 30 minutes either way (remote or in person). It is about 20minutes for the test, 5 minutes for questions, and a 5 minute buffer period.

Last week I sent you Google calendar invites with the time slots, so you are free to add the user to the correct slot.!

Please let me know if that answers your question.

**Comment by MentalSnapp** [comment]
Is there space in the office all of them? Otherwise I think we should do it
by remote. I didn’t see the invite can you forward?

**Comment by teenie-quaggard** [comment]
It kind of depends on where we are on Thursday. Our contract still has not been signed, so if they would like to come in, I would recommend that we try meeting in a nearby cafe. Otherwise it should be fine remotely.

**Comment by teenie-quaggard** [comment]
Hi @MentalSnapp - I've resent the invites so you should now have time slots between 9-10:30am! Please let me know if you have any issues.

**Comment by MentalSnapp** [comment]
Let’s do it remotely.

I’m going to try to get there for 8.45.

See you then!

**Comment by teenie-quaggard** [comment]
@MentalSnapp Ok, see you then! We still do not have the contract signed, so it may be best if we just meet at Starbucks nearby. I will also aim to be there around 8:45am. Here is the address:
Wells Terrace, London, N4, England
